src/tools/rest-frontend/public/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html ng-app="elektra.rest.angular">
+
+    <head>
+
+        <!-- Application base path (i.e. subfolder of URL where application is reachable, mostly /) -->
+        <base href="/" />
+
+        <!-- Meta information -->
+        <title>ElektraInitiative</title>
+        <meta charset="utf-8">
+        <meta name="keywords" content="elektra, rest, repository, kdb, configuration, config">
```

Comment:
No idea. The only place where they are actually relevant is the index.html. In the package.json they make only sense if we distribute the package via npm as standalone, but I doubt that this makes any sense at all.